x-pack/plugins/secops/public/components/flyout/index.tsx

```diff
@@ -55,13 +54,17 @@ export const Text = styled(EuiText)`
   z-index: 3;
 `;
 
+const Visible = styled.div<{ show: boolean }>`
```

Comment:
this is a nice way of cleaning up conditional styles

Reply:
Yeah, below it I figured out that you can destructure the `props` as well.  I didn't cleanup the rest of the code base on that tid-bit, but we can do destructuring within the `props` for styled components which is good news to keeping the syntax noise ratio lower.